modules/configuring-pipelines-as-code-for-a-git-repository-hosting-service-provider.adoc

```diff
@@ -0,0 +1,21 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="configuring-pipelines-as-code-for-a-git-repository-hosting-service-provider_{context}"]
+= Configuring {pac} for a Git repository hosting service provider 
+
+[role="_abstract"]
+After installing {pac}, cluster administrators can configure a Git repository hosting service provider. Currently, the following services are supported:
+
+* Github App
+* Github Webhook
+* Bitbucket Server
+* Bitbucket Cloud
+
+[NOTE]
+====
+GitHub App is the recommended service for using Pipelines as Code.
```

Comment:
It might be helpful to explain why that is and the advantages of GitHub App over the other methods

Reply:
@sm43, @chmouel - Please help me out here with some explanation about the advantages, as asked by Siamak.

Reply:
@siamaksade : I have the following abstract for the module describing the procedure for GitHub apps:

*GitHub Apps act as a point of integration with {pipelines-title} and bring the advantage of Git-based workflows to Tekton Pipelines. Cluster administrators can configure a single GitHub App for all users of the cluster. For GitHub Apps to work with Pipelines as Code, ensure that the webhook of the GitHub App points to the Pipelines as Code controller route (or ingress endpoint) that listens for GitHub events.*

In the finished content, it will appear just below the note for which you have commented. 
Does that solve the problem? Otherwise, we might repeat the same thing in proximity.